source/dataman/datasource/AMDataSourceSeriesData.cpp

```diff
@@ -156,7 +156,7 @@ void  AMDataSourceSeriesData::onDataChanged(const AMnDIndex &start, const AMnDIn
 
 void AMDataSourceSeriesData::onAxisValuesChanged()
 {
-	QVector<AMNumber> axisData = QVector<AMNumber>(axisSize_, 0);
+    QVector<double> axisData = QVector<double>(axisSize_, 0);
 	source_->axisValues(0, 0, axisSize_-1, axisData.data());
```

Comment:
Wait! You can get access to the underlying data in array form like this? I went through the rigmarole of creating a whole template class to safely handle arrays. :(
I'll certainly use this in future.